common.sh

```diff
@@ -171,6 +171,10 @@ if grep -q "Red Hat Enterprise Linux release 8" /etc/redhat-release 2>/dev/null
     export RHEL8="True"
 fi
 
+if grep -q "CentOS Linux release 8" /etc/redhat-release 2>/dev/null; then
+    CENTOS8="True"
```

Comment:
1. Do you need `export` here? RHEL8 uses it.

2. Cannot we reuse the same RHEL8 variable? Do you expect any differences?

Reply:
yes this should be export

It could use the same variable but it's probably OK to add a new one - currently the only differences are the repos enabled in metal3-dev-env (which figures out the OS version without this variable), but I suppose we might find something which works different on centos.

The goal is to remove all these conditionals though, when we remove support for centos7, at that point we'd only special case things which really were different between rhel and centos (ideally nothing).